src/overlays/gamestates/ovl_select/z_select.h

```diff
@@ -3,12 +3,39 @@
 
 #include "global.h"
 
+struct SelectContext;
+
 void Select_Init(GameState* thisx);
 void Select_Destroy(GameState* thisx);
 
+typedef void (*Select_LoadFunc)(struct SelectContext*, u32, s32);
+
 typedef struct {
-    GameState state;
-    char unk_A4[0x1DC];
+    /* 0x00 */ char* name;
+    /* 0x04 */ Select_LoadFunc loadFunc;
+    /* 0x08 */ s32 entranceIndex;
+} SceneSelectEntry; // size = 0xC
+
+typedef struct SelectContext {
+    /* 0x000 */ GameState state;
+    /* 0x0A8 */ View view;
+    /* 0x210 */ s32 count;
+    /* 0x214 */ SceneSelectEntry* scenes;
+    /* 0x218 */ s32 currentScene;
+    /* 0x21C */ s32 pageDownIndex; // Index of pageDownStops
+    /* 0x220 */ s32 pageDownStops[7];
+    /* 0x23C */ char unk_23C[0xC];
+    /* 0x248 */ s32 opt;
```

Comment:
I'd note that this is the name the menu uses and what its actual use is, since it's not obvious in the struct.